excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreator.java

```diff
@@ -225,16 +228,16 @@ public String createErrorMessage(final CaseTypeEntityInvalidUserRoleValidationEr
         return newMessageIfDefinitionExists(error,
             error.getCaseTypeUserRoleEntity(),
 
-            def -> CaseRoleServiceImpl.isCaseRole(defaultString(def.getString(ColumnName.USER_ROLE)))
+            def -> CaseRoleServiceImpl.isCaseRole(defaultString(def.getString(ColumnName.ACCESS_PROFILE)))
                 ? String.format(
                 "Invalid case role '%s' in %s tab for case type '%s'. "
                     + DEFINED_IN_CASE_ROLES_SHEET_VERIFICATION_MESSAGE,
-                def.getString(ColumnName.USER_ROLE),
+                def.getString(ColumnName.ACCESS_PROFILE),
                 def.getSheetName(),
                 error.getAuthorisationValidationContext().getCaseReference()) :
                 String.format(
-                    "Invalid idam role '%s' in %s tab for case type '%s'",
-                    def.getString(ColumnName.USER_ROLE),
+                    "Invalid IDAM access profile '%s' in %s tab for case type '%s'",
```

Comment:
Remove the word IDAM

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f